src/__tests__/Credentials-test.ts

```diff
@@ -679,9 +697,125 @@ describe('txRequest()', () => {
       callbackUrl,
       label: 'Update Status'
     })
-    const verified = await verifyJWT(jwt)
+    const verified = await verifyJWT(jwt, { resolver: credentials.resolver })
     expect(verified.payload.net).toEqual(networkId)
     expect(verified.payload.callback).toEqual(callbackUrl)
     expect(verified.payload.label).toEqual('Update Status')
   })
 })
+
+describe('issueVerifiableCredential', () => {
+  let credentials: Credentials
+
+  const vcPayload: any = {
+    sub: 'did:ethr:0x435df3eda57154cf8cf7926079881f2912f54db4',
+    nbf: Date.now() / 1000,
+    vc: {
+      '@context': ['https://www.w3.org/2018/credentials/v1'],
+      type: ['VerifiableCredential'],
+      credentialSubject: {
+        degree: {
+          type: 'BachelorDegree',
+          name: 'Baccalauréat en musiques numériques'
+        }
+      }
+    }
+  }
+  it('vc test', async () => {
+    credentials = new Credentials({ privateKey, did })
+    const jwt = await credentials.issueVerifiableCredential(vcPayload)
+    const decoded = await verifyCredential(jwt, credentials.resolver)
+    expect(decoded.payload.vc).toEqual(vcPayload.vc)
+  })
+})
+
+describe('verifyPresentation', () => {
+  let credentials: Credentials
+
+  const vcPayload: any = {
+    sub: 'did:ethr:0x435df3eda57154cf8cf7926079881f2912f54db4',
+    nbf: 1562950282,
+    vc: {
+      '@context': ['https://www.w3.org/2018/credentials/v1'],
+      type: ['VerifiableCredential'],
+      credentialSubject: {
+        degree: {
+          type: 'BachelorDegree',
+          name: 'Baccalauréat en musiques numériques'
+        }
+      }
+    }
+  }
+
+  it('verifyPresentation', async () => {
+    credentials = new Credentials({ privateKey, did })
+    const vcJwt = await credentials.issueVerifiableCredential(vcPayload)
+
+    const vpPayload: any = {
+      vp: {
+        '@context': ['https://www.w3.org/2018/credentials/v1'],
+        type: ['VerifiableCredential'],
+        verifiableCredential: [vcJwt]
+      }
+    }
+
+    const issuer: Issuer = { did: credentials.did, signer: credentials.signer }
+    const vpJwt = await createPresentation(vpPayload, issuer)
+
+    const result = await credentials.verifyPresentation(vpJwt)
+    expect(result.payload.vp).toEqual(vpPayload.vp)
```

Comment:
this or another test should also check for the other fields in the payload, not just `.vp`